### PR TITLE
Update formatting of office hours doc

### PR DESCRIPTION
**Pull request by mrbobbytables**
This is a no-op. No content changes were made, just formatting.

### CODE REVIEW

File: events/office-hours.md [hunk 1]
```diff
@@ -62,41 +99,86 @@ This information is for the host/panel to prep the show.
...
+  - This is a judgement-free zone, everyone had to start from somewhere so please
+    help out your buddy by having a supportive environment in the channel.
+  - While we will do our best to answer your questions the panel doesn’t have
+    access to your cluster, so live debugging is off topic, but we will do our
+    best to get you moving down the next step.
+  - Panelists, you’re encouraged to expand on answers with your experiences and
+    pro-tips.
+  - Audience, you can help by pasting in URLs to official docs, blogs, or
+    anything that might be relevant to the topic at hand.
+  - Post your questions on [discuss.kubernetes.io]
```
Comment (coderanger) [inline events/office-hours.md:144]: No period after the text, unlike all other lines.

File: events/office-hours.md [hunk 2]
```diff
@@ -62,41 +99,86 @@ This information is for the host/panel to prep the show.
...
+      for each event.
+- [ ] Monday before: Post "Office hours is this Wednesday" with the links to
+      everything in [#kubernetes-users], [#office-hours], and [#kubernetes-novice].
+- [ ] Monday before: Post to the general section of discuss.k8s.io with a link
+      to the show and a link to this github page. Tag the post with [#office-hours]
+- [ ] Monday before: Tweet "Office hours is this Wednesday" with the links to
+      everything, send a mail to social@cncf.io to RT your tweet.
+- [ ] -1h before show: Tweet again, repost "Office hours is going live in ~1h"
+      with the links to everything in [#kubernetes-users], [#office-hours], and
+      [#kubernetes-novice]
```
Comment (coderanger) [inline events/office-hours.md:114]: Also missing end of line period.

Comment (mrbobbytables) [reply events/office-hours.md]: done 👍

File: events/office-hours.md (hunk 1 above)
Comment (mrbobbytables) [reply events/office-hours.md]: done 👍

**Pull request merged**